client/app/components/dev_components/testdata.js

```diff
@@ -3,9 +3,9 @@ export default {
   name: 'Test',
   location: {
     address: '123 Main St',
-    coordinate: {
-      latitude: 0,
-      longitude: 0,
-    },
+  },
+  coordinates: {
```

Comment:
I like this change, but feel that as a result, location is no longer necessary as a key, and can be replaced with address. Do you agree?